cmake/pcl_targets.cmake

```diff
@@ -110,8 +110,10 @@ macro(PCL_SUBSYS_DEPEND _var)
   if(ARGS_OPT_DEPS)
     SET_IN_GLOBAL_MAP(PCL_SUBSYS_OPT_DEPS ${_name} "${ARGS_OPT_DEPS}")
   endif()
+
   GET_IN_MAP(subsys_status PCL_SUBSYS_HYPERSTATUS ${_name})
-  if(${_var} AND (NOT ("${subsys_status}" STREQUAL "AUTO_OFF")))
```

Comment:
Isn't this set by `PCL_SUBSYS_OPTION`? So it should exist?
As I understand this check, if build is `FALSE`, it is unnecessary to check the dependencies

Reply:
Must have written wrong when I tested it. It exist indeed.